include/color_spinor_field_order.h

```diff
@@ -861,28 +861,31 @@ namespace quda {
   }
 
   __device__ __host__ inline void save(const RegType v[length], int x, int parity=0) {
-    RegType scale = 0.0;
     RegType tmp[length];
 
     if (sizeof(Float)==sizeof(short) || sizeof(Float)==sizeof(char)) {
```

Comment:
Use `if_fixed` trait?

Reply:
Again, were you looking at an old file somehow here?